hack/feature-deploy.sh

```diff
@@ -38,9 +54,9 @@ do
     set +e
     # be verbose on last iteration only
     if [[ $iterations -eq $((max_iterations - 1)) ]] || [[ -n "${VERBOSE}" ]]; then
-      ${OC_TOOL} apply -k "$feature_dir"
+      $KUSTOMIZE_BIN build "$feature_dir" | ${OC_TOOL} apply -f -
```

Comment:
Can you add a TODO to revert this to oc once the kustomize feature is in?

Reply:
that TODO comment can link to
https://github.com/kubernetes/kubectl/issues/818
when that it closed, it'll get merged in the next oc too